source/overloading.tex

```diff
@@ -3084,8 +3084,6 @@
     \terminal{<<       >>       <<=      >>=      ++       --       ,}\br
 \end{bnf}
 \begin{note}
-The last two operators are function call\iref{expr.call}
-and subscripting\iref{expr.sub}.
```

Comment:
Maybe move this to the end of the note, and change "the last" to "the latter"? I think it's useful to mention in passing here what `operator()` and `operator[]` mean.